sw/device/tests/sram_ctrl_scrambled_access_test.c

```diff
@@ -38,16 +58,16 @@ static const sram_ctrl_data_t kRamTestPattern2 = {
  * which prevents a false-negative in case of the value of the memory address
  * under test is already initialised to one of the patterns.
  */
-static void test_ram_write_read_pattern(const sram_ctrl_t *sram) {
-  // Write first pattern to the start and the end of RAM.
-  sram_ctrl_write_to_ram(sram, &kRamTestPattern1);
-  CHECK(sram_ctrl_read_from_ram_eq(sram, &kRamTestPattern1),
+static void test_ram_write_read_pattern(void) {
+  // Write first pattern to the start of SRAM, and read it out.
+  sram_ctrl_testutils_write(kSramStartAddr, &kRamTestPattern1);
+  CHECK(sram_ctrl_testutils_read_check_eq(kSramStartAddr, &kRamTestPattern1),
         "Read-Write first pattern failed");
 
-  // Write second pattern to the start and the end of RAM.
-  sram_ctrl_write_to_ram(sram, &kRamTestPattern2);
-  CHECK(sram_ctrl_read_from_ram_eq(sram, &kRamTestPattern2),
-        "Read-Write second pattern failed");
+  // Write second pattern to the start of SRAM, and read it out.
+  sram_ctrl_testutils_write(kSramStartAddr, &kRamTestPattern2);
+  CHECK(sram_ctrl_testutils_read_check_eq(kSramStartAddr, &kRamTestPattern2),
+        "Read-Write first pattern failed");
```

Comment:
The comment says second pattern, but the log says first. I think the log message is wrong, right?

Reply:
OOps, missed this comment, will do it now.

Reply:
This is from the other #9138, I will fix it up there.